packages/kbn-esql-validation-autocomplete/src/definitions/types.ts

```diff
@@ -137,7 +137,7 @@ export interface FunctionDefinition {
        * we can't check the return value of a function to see if it
        * matches one of the options prior to runtime.
        */
-      literalOptions?: string[];
+      acceptedValues?: string[];
```

Comment:
Not related to the PR, but a good change IMO

Reply:
Def makes more sense comparing to the other 👍